cmd/site-replication.go

```diff
@@ -2382,6 +2401,9 @@ func (c *SiteReplicationSys) RemovePeerCluster(ctx context.Context, objectAPI Ob
 	st = madmin.ReplicateRemoveStatus{
 		Status: madmin.ReplicateRemoveStatusSuccess,
 	}
+	if !c.isEnabled() {
+		globalSiteReplicatorCred = auth.Credentials{}
```

Comment:
Is this needed?

Reply:
can omit this since the config is disabled anyways